funcbench/main.go

```diff
@@ -99,35 +124,46 @@ The default is 2 hours (2h).`).Default("2h").Duration()
 				err error
 			)
 
-			if *gitHubPRNumber == 0 {
-				env, err = newLocalEnv(environment{
-					logger:        logger,
-					benchFunc:     *benchFunc,
-					compareTarget: *compareTarget,
-					home:          os.Getenv("HOME"),
-				})
+			// Setup Environment.
+			e := environment{
+				logger:        logger,
+				benchFunc:     cfg.benchFuncRegex,
+				compareTarget: cfg.compareTarget,
+			}
+			if cfg.ghPr == 0 {
+				// Local Mode.
+				env, err = newLocalEnv(e)
 				if err != nil {
-					return errors.Wrap(err, "new env")
+					return errors.Wrap(err, "environment creation error")
 				}
-				logger.Printf("funcbench start [Local Mode]: Benchmarking current version versus %q for benchmark funcs: %q\n", *compareTarget, *benchFunc)
 			} else {
-				if *owner == "" || *repo == "" {
-					return errors.New("funcbench in GitHub Mode requires --owner and --repo flags to be specified")
+				// Github Mode.
+				ghClient, err := newGitHubClient(ctx, cfg.owner, cfg.repo, cfg.ghPr, cfg.dryrun)
+				if err != nil {
+					return errors.Wrapf(err, "could not create github client")
```

Comment:
ditto

Reply:
and everywhere else. It's clear we could not create ... it's already  wrapping error (: So it's just a spam